src/index.css

```
@@ -5,6 +5,344 @@
+  --chart-5: var(--tds-purple-500);
+  --sidebar: var(--tds-grey-background);
+  --sidebar-foreground: var(--tds-grey-opacity-800);
+  --sidebar-primary: var(--tds-blue-500);
+  --sidebar-primary-foreground: var(--tds-white);
+  --sidebar-accent: var(--tds-grey-opacity-100);
+  --sidebar-accent-foreground: var(--tds-grey-opacity-800);
+  --sidebar-border: var(--tds-grey-opacity-200);
+  --sidebar-ring: var(--tds-blue-500);
+  --font-sans: "Pretendard", sans-serif;
+  --font-serif: ui-serif, Georgia, Cambria, "Times New Roman", Times, serif;
+  --font-mono: Geist Mono, ui-monospace, monospace;
```

Stylelint (17.11.1)

[error] Expected "Geist" to be "geist" (value-keyword-case)

(value-keyword-case)